action.yml

```diff
@@ -27,6 +27,18 @@ inputs:
     description: "The directory where your Gatsby source files are at. `gatsby build` will run from this directory."
     required: false
     default: "."
+  commit-message:
+    description: "The commit message used for pushing changes to the deploy branch. Use the identifier {sha} to replace with the commit SHA."
```

Comment:
I really like the initiative here to allow to still include the `sha` but I would like to introduce this in a simpler form by only allowing a string message without the identifier. We could always introduce that in the future if there's a need. 😃